src/plugins/plot/MctPlot.vue

```diff
@@ -1190,28 +1190,42 @@ export default {
         selectNearbyAnnotations(event) {
             // need to stop propagation right away to prevent selecting the plot itself
             event.stopPropagation();
-            if (!this.annotationViewingAndEditingAllowed || this.annotationSelections.length) {
+
+            const nearbyAnnotations = this.gatherNearbyAnnotations();
+
+            if (this.annotationViewingAndEditingAllowed && this.annotationSelections.length) {
```

Comment:
Let's just put 
```js
if (!this.annotationViewingAndEditingAllowed) {
    return;
}
```

after `stopPropagation()` and remove the redundant check from the other conditions?

Reply:
No, that would be incorrect. If annotations are not allowed, we should fallback to plot selection - `this.selectPlot()`

Reply:
so it's okay to do `this.selectPlot()` regardless of whether `this.annotationViewingAndEditingAllowed` is true or false?

I'm a bit confused here